site/content/en/installation/binaries/_index.md

```diff
@@ -16,7 +16,15 @@ Download pre-compiled binaries.
 | [Windows (x64)][windows]
 
 ```sh
-# For linux/mac
+# Download binaries using gsutil:
+# Linux (x64)
+gsutil cp gs://kpt-dev/latest/linux_amd64/kpt .
```

Comment:
If we download kpt to the current directory then can we also update the `kpt version` command to be `./kpt version` or add a note about moving the kpt binary into the user's path?

Reply:
Done

Reply:
Why gsutil? These are public artifacts, so we can use curl.

Reply:
Windows doesn't have `curl` by default.

Reply:
Updated - use `curl` and removed instructions for Windows